app/common/constants/computedStyles.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const computedStyles = {
```

Comment:
oh, almost forgot. We should comment this file and the less files to make sure they don't get out-of-sync